src/gui/mainwindow.cpp

```diff
@@ -38,9 +38,15 @@ MainWindow::MainWindow(QWidget* parent)
 
     connect(canDevice, &CanDevice::frameReceived, canRawView, &CanRawView::frameReceived);
     connect(canDevice, &CanDevice::frameSent, canRawView, &CanRawView::frameSent);
-
+    connect(ui->actionstart, SIGNAL(triggered(bool)), canRawView, SLOT(simulationStarted(void)));
```

Comment:
Try to keep consistency while adding similar code to already existing one. "connect" version with function pointers is better as it checks if signals/slots exists during compilation rather than runtime.